Python/Python.sublime-syntax

```diff
@@ -1782,7 +1783,6 @@ contexts:
       scope: keyword.control.flow.else.inline.python
 
   target-list:
-    - include: line-continuation-or-pop
```

Comment:
And this is likely the change that cause the test breakage.

It would be better to change the usage of it in the inline for context because removing the include here will break other cases (that aren't being tested currently, it seems).

https://github.com/Thom1729/Packages/blob/4a1543f14949593fbde7553a7361543edc92f0a1/Python/Python.sublime-syntax#L1777

Proposal: outsource all items from `target-list` but this one exclude into a `target-list-inline` context and include that from the inline for context.

Reply:
It looks to me like the cases that need `line-continuation-or-pop` (in `block-statements`) are including it manually. The only place that doesn't is `inline-for`, which doesn't want it. I believe the lookahead fix in `allow-unpack-operators` led to the test failure.

Reply:
Looks like it indeed. This change is fine then.